airflow-core/src/airflow/ui/src/i18n/locales/nl/common.json

```
@@ -0,0 +1,78 @@
+{
+  "defaultToGraphView": "Standaard grafiekweergave",
+  "defaultToGridView": "Standaard rasterweergave",
+  "logout": "Uitloggen",
+  "switchToDarkMode": "Schakel over naar Donkere Modus",
+  "switchToLightMode": "Schakel over naar Lichte Modus",
+  "timezone": "Tijdzone",
+  "user": "Gebruiker",
+  "language": {
+    "chinese": "Traditioneel Chinees",
+    "english": "Engels",
+    "dutch": "Nederlands",
+    "select": "Selecteer Taal"
+  },
+  "nav": {
+    "home": "Home",
+    "assets": "Assets",
+    "browse": "Verken",
+    "admin": "Beheer",
+    "docs": "Documentatie",
+    "plugins": "Plugins"
+  },
+  "browse": {
+    "auditLog": "Audit Log",
+    "xcoms": "XComs"
+  },
+  "admin": {
+    "Variables": "Variabelen",
+    "Pools": "Pools",
+    "Providers": "Providers",
+    "Plugins": "Plugins",
+    "Connections": "Connecties",
+    "Config": "Configuratie"
+  },
+  "timeRange": {
+    "duration": "Duur",
+    "lastHour": "Laatste Uur",
+    "last12Hours": "Laatste 12 Uur",
+    "last24Hours": "Laatste 24 Uur",
+    "pastWeek": "Laatste Week"
+  },
+  "docs": {
+    "documentation": "Documentatie",
+    "githubRepo": "GitHub Repo",
+    "restApiReference": "REST API Referentie"
+  },
+  "states": {
+    "queued": "Wachtend",
+    "running": "Lopend",
+    "success": "Succesvol",
+    "failed": "Mislukt",
+    "skipped": "Overgeslagen",
+    "removed": "Verwijderd",
+    "scheduled": "Gepland",
+    "restarting": "Herstartend",
+    "up_for_retry": "Wachtend op een nieuwe poging",
+    "up_for_reschedule": "Wachtend op herplanning",
+    "upstream_failed": "Upstream Mislukt",
```

I suggest we stick with one way of casing (titlecase vs sentencecase). Given the 2 sentences above are in sentencecase, how about:

```suggestion
    "upstream_failed": "Upstream mislukt",
```

---

I totally agree! Initially I just copied the casing, but that isn't proper Dutch. I'll change it to sentencecase later today